util/printchplenv

```diff
@@ -111,6 +111,9 @@ def print_mode(mode='list', anonymize=False):
     hwloc = chpl_hwloc.get()
     print_var('CHPL_HWLOC', hwloc, mode, 'hwloc', ('runtime',))
 
+    jemalloc = chpl_jemalloc.get()
+    print_var('CHPL_JEMALLOC', jemalloc, mode, 'jemalloc', ('runtime',))
```

Comment:
We could, but only for certain configurations (e.g. not in printchplenv --make). I'm half a mind to leave it alone though (like CHPL_HWLOC prints with or without qthreads). The other precedent is CHPL_COMM_SUBSTRATE etc which only print with CHPL_COMM != none.

Is it obvious to you what we should do here?